src/components/Greetings.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getGreeting } from '../redux/greetings';
+
+const Greeting = () => {
+  const { greeting } = useSelector((state) => state.greeting);
+  const { status } = useSelector((state) => state.greeting);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getGreeting());
+  }, []);
+
+  let screenMessage;
+  if (status === 'success') {
+    screenMessage = greeting.message;
+  }
+  if (status === 'pending') {
+    screenMessage = 'loading...';
+  }
+
+  return (
+    <>
+      <h1>Refresh the Page to See My Random Greetings 😃</h1>
+      <span>.......................................................</span>
+      <p>{screenMessage}</p>
+      <span>.......................................................</span>
+    </>
+  );
+};
+
```

Comment:
- [ ] Kindly make sure the text of the message is correctly displayed on the screen as required for the project. Right now, your greeting component displays nothing instead the API has some greetings message. I even try to debug by using `console.log()` but I didn't get anything. 

![review](https://user-images.githubusercontent.com/83042742/208277960-1f8a3bdf-4bd4-48dc-864c-29a4a9fa8bea.gif)